### PR TITLE
fix js-beautify version to 1.6.14, since js-beautify@1.7.0 is fucked

### PR DESCRIPTION
**Pull request by zyy7259**
vue-loader specify the version of js-beautify as ^1.6.14, which will allow the installation of js-beautify@1.7.0.

But package js-beautify@1.7.0 is not published correctly: https://github.com/beautify-web/js-beautify/issues/1247.

Temporarily fix its version to 1.6.14 can solve this problem.

### CODE REVIEW

**Comment by lmj0011** [comment]
this should solve issue I was having here:

https://github.com/beautify-web/js-beautify/issues/1247#issuecomment-330159973

**Comment by LinusBorg** [comment]
I'm npot sure we should release a patch that freezes the version like that. people installing this version will be stuck with it.

This might be a solution if `js-beautify` doesn't resolve the problem in a timely manner, but we should wait out a little because the fix they need is very easy, the npm package is missing a few files they have to add again.

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ test/output
 docs/_book
 .DS_Store
 .idea
+.history
```
Comment (LinusBorg) [inline .gitignore]: unrelated change, should be removed.

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "consolidate": "^0.14.0",
     "hash-sum": "^1.0.2",
-    "js-beautify": "^1.6.14",
+    "js-beautify": "1.6.14",
```
Comment (LinusBorg) [inline package.json]: we should do this once we see that `js-beautify` fails to actually fix it. We should wait a while.

**Comment by Oliboy50** [comment]
nice try but => https://github.com/vuejs/vue-loader/commit/97bd432bba4350988bfd0db6d7a1a514b66a9826

**Comment by bitwiseman** [comment]
@LinusBorg - The install is fixed. It is up to you if you want to switch to prettier permanently.

File: package.json (hunk 2 above)
Comment (bitwiseman) [reply package.json]: Already fixed.  1.7.3 works.  The `js-beautify` project does use semantic versioning so `~1.7.3` would also be reasonable.

**Comment by LinusBorg** [comment]
I will close this for now, will talk with the others about this.
